test/fail_compilation/fail137.d

```diff
@@ -1,5 +1,11 @@
 // https://issues.dlang.org/show_bug.cgi?id=751
 // Compiler segfault on template expansion
+/*
+TEST_OUTPUT:
+---
+fail_compilation/fail137.d(28): Error: `printf` is not defined, perhaps `import core.stdc.stdio;` is needed?
```

Comment:
This error does not match the comment above (`Compiler segfault on template expansion`)

The test should be updated to add a stub `printf()` declaration.

Reply:
#9232